src/ImageSharp.Web/Middleware/ImageSharpMiddleware.cs

```diff
@@ -50,14 +50,8 @@ public class ImageSharpMiddleware
         /// <summary>
         /// Used to temporarily store cache resolver reads to reduce the overhead of cache lookups.
         /// </summary>
-        private static readonly ConcurrentTLruCache<string, IImageCacheResolver> CacheResolverLru
-            = new ConcurrentTLruCache<string, IImageCacheResolver>(1024, TimeSpan.FromMinutes(5));
-
-        /// <summary>
-        /// Used to temporarily store cache metadata reads to reduce the overhead of cache lookups.
-        /// </summary>
-        private static readonly ConcurrentTLruCache<string, ImageCacheMetadata> CacheMetadataLru
-            = new ConcurrentTLruCache<string, ImageCacheMetadata>(1024, TimeSpan.FromMinutes(5));
+        private static readonly ConcurrentTLruCache<string, (IImageCacheResolver, ImageCacheMetadata)> CacheResolverLru
```

Comment:
I would se you either need to name these tuple values or create a custom struct that encapsulates these two fields.

Reason is it will remove the horrid use of `foo.Item1` & `foo.Item2` field access. with well named fields the follow on code would be significantly more readable.

ie in some code you have later you have `cachedImageResolver.Item2.SourceLastWriteTimeUtc` when reading in isolation there is nothing to hint at what `Item2` means in that line

Reply:
Agreed.

Reply:
Done. 

Hmm, updated the submodule as suggested and the code coverage complaints have skyrocketed :(